content/millennium/r4.md

```diff
@@ -414,6 +414,17 @@ Where possible, the FHIR<sup>®</sup> standard strives to use appropriate HTTP v
  `POST` | Used for creating resources.
  `PUT`  | Used for updating resources.
 
+### HTTP method Override
+If your firewall rules do not support HTTP methods like PUT, PATCH or DELETE, the request can be made through POST using one of the following:
```

Comment:
This wouldn't be used for firewall issues. This would be used if the client library doesn't support those methods.

Reply:
How about i make it more generic? something like
 _If support fo HTTP methods like PUT, PATCH or DELETE is unavailable, the request can be.._

Reply:
https://github.com/cerner/fhir.cerner.com/pull/263/commits/76e9210f64aeffa0baa8b15143892df2251195eb